build.gradle.kts

```diff
@@ -63,6 +65,12 @@ allprojects {
 
 disableDistTasks()
 
+tasks.test {
+    useJUnitPlatform()
```

Comment:
jvm tests are already configured with this call, see https://github.com/JetBrains/koog/blob/develop/buildSrc/src/main/kotlin/ai/grazie/gradle/tests/JvmTestConfigure.kt